Modules/_bisectmodule.c

```diff
@@ -281,12 +295,54 @@ expensive comparison operations, this can be an improvement over the more\n\
 common approach.\n");
 
 
+static int
+bisect_traverse(PyObject *module, visitproc visit, void *arg)
+{
+    bisect_state *state = get_bisect_state(module);
+    Py_VISIT(state->str_insert);
```

Comment:
I think you can drop the traverse method. cc. @vstinner

Reply:
@erlend-aasland 
Yeah I forgot the fact: https://github.com/python/cpython/pull/31608